components/Tooltip.js

```diff
@@ -22,23 +23,36 @@ class Tooltip extends Component {
     showIcon: true,
     iconName: 'info--glyph',
     iconDescription: 'tooltip',
+    positionThreshold: 5,
```

Comment:
@chrisdhanaraj there is a property named triggerPosition in the state, which indicates the position of the tooltip content, right? but this property will only been calculate when component did mounted, so if a tooltip component in dialog and the component's position is updated, then the tooltip content will not been updated.